x-pack/plugins/fleet/public/components/enrollment_instructions/install_section.tsx

```diff
@@ -44,6 +46,7 @@ export const InstallSection: React.FunctionComponent<Props> = ({
         linuxRpmCommand={installCommand.rpm}
         k8sCommand={installCommand.kubernetes}
         hasK8sIntegration={isK8s === 'IS_KUBERNETES' || isK8s === 'IS_KUBERNETES_MULTIPAGE'}
+        typeOfCSPIntegration={isCSP}
```

Comment:
in some parts we are passing just the type, like that: 

```js
typeOfCSPIntegration={isCSP}
```

while in others we pass the whole object,

I think it would be better to always pass the whole object like that:

```js
cloudSecurityIntegration={cloudSecurityIntegration}
```

So we keep consistency, avoid confusion and we can reuse the types in other places.